flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java

```diff
@@ -49,6 +49,14 @@
 			.defaultValue(-1)
 			.withDescription("The port where the application master RPC system is listening.");
 
+	/**
+	 * The vcores used by YARN application master.
+	 */
+	public static final ConfigOption<Integer> APP_MASTER_VCORES =
+		key("yarn.appmaster.vcores")
+		.defaultValue(1)
+		.withDescription("The number of virtual cores (vcores) used by YARN application master.");
```

Comment:
I guess we need to regenerate the configuration documentation because we added a new config option: `mvn package -Dgenerate-config-docs -pl flink-docs`.

Reply:
I have tried, but it does not take effect. In addition, it seems all the generated HTML documentation are not tracked by git.

Reply:
You first need to install `flink-yarn`. I'll do it while merging.